app/MMB/src/scenes/tripServices/transportationMenuItem/TransportLocationItem.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import idx from 'idx';
+
+import LocationPopupButton from '../LocationPopupButton';
+import type { LocationItem as LocationItemType } from './__generated__/TransportLocationItem.graphql';
+
+type Props = {|
+  +data: LocationItemType,
+  +onPress: ({
+    location: { lat: number, lng: number },
+    whitelabelURL: string,
+  }) => void,
+  +whitelabelURL: string,
+|};
+
+class LocationItem extends React.Component<Props> {
+  onPress = () => {
+    const lat = idx(this.props.data, _ => _.location.lat) || 0;
```

Comment:
this is not the best fallback, https://en.wikipedia.org/wiki/Null_Island

Reply:
What do you suggest I take then? It really depends on each location, so it's hard to choose one that would make sense? Brno as Kiwi was born in Brno?

Reply:
Maybe show an alert say there is something wrong 🤷‍♂️  better than sending coordinates in the middle of the ocean, don't you think?